Vagrantfile

```diff
@@ -44,19 +44,39 @@ Vagrant.require_version ">= #{vconfig['drupalvm_vagrant_version_min']}"
 
 ensure_plugins(vconfig['vagrant_plugins'])
 
+# Local host updater settings
+begin
+  disable_hostsupdater = vconfig.fetch('disable_hostsupdater')
+rescue KeyError
```

Comment:
Since this is in the defaults, it will always be defined—we can just use the line:

```
disable_hostsupdater = vconfig.fetch('disable_hostsupdater')
```

Reply:
Thx for the suggestion. It's changed.